Readme.md

```diff
@@ -243,6 +255,18 @@ _How to implement security features with existing libraries._
 * `openssl`
 * `libsodium`
 * `wolfssl`
+* Mbed TLS
+</details>
+
+<details><summary><b>Binaries</b></br>
```

Comment:
I was unsure about this as I first thought it rather to be related to application development... But you're right, it's quite Linux-specific.

Reply:
Since all of us will encounter these things at one point, it might be a good idea to have a little foundation in it. 😉